src/Commands/StatusCommand.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+namespace UKFast\HealthCheck\Commands;
+
+use Illuminate\Console\Command;
+use UKFast\HealthCheck\Facade\HealthCheck;
+
+class StatusCommand extends Command
+{
+    protected $signature = 'health-check:status';
+
+    protected $description = 'Check health status';
+
+    public function handle()
+    {
+        $problems = [];
+        $isOkay = true;
+        /** @var \UKFast\HealthCheck\HealthCheck $check */
+        foreach (HealthCheck::all() as $check) {
+            $status = $check->status();
+
+            if (!$status->isOkay()) {
+                $problems[] = [$check->name(), $status->name(), $status->message()];
+
+                if ($isOkay) {
+                    $isOkay = false;
+                }
+            }
+        }
+
+        if (!$isOkay) {
```

Comment:
Just a suggestion, but instead of keeping track of an `isOkay` variable, you could check if `empty($problems)`

Reply:
Made a little refactoring